### ISSUE TITLE
Error with imageio 2.3.0

### ISSUE THREAD

**Issue by 4383** [sourcerer-io/anishot#8]
Hi,

With imageio 2.3.0 the following error occur:
```shell
$ anishot --inp ~/Images/anishot.png --out /tmp/output.git --h 200
Traceback (most recent call last):
  File "/home/herve/.local/share/virtualenvs/anishot-2ESls-8g/bin/anishot", line 10, in <module>
    sys.exit(main())
  File "/home/herve/dev/anishot/anishot/__main__.py", line 124, in main
    duration=list(map(lambda f: f[1], frames)))
  File "/home/herve/.local/share/virtualenvs/anishot-2ESls-8g/local/lib/python2.7/site-packages/imageio-2.3.0-py2.7.egg/imageio/core/functions.py", line 320, in mimwrite
    writer = get_writer(uri, format, 'I', **kwargs)
  File "/home/herve/.local/share/virtualenvs/anishot-2ESls-8g/local/lib/python2.7/site-packages/imageio-2.3.0-py2.7.egg/imageio/core/functions.py", line 170, in get_writer
    'in mode %r' % mode)
ValueError: Could not find a format to write the specified file in mode 'I'
```

I doesn't try with other version of imageio...

What's the current version of imageio that you use in your example on README?

**Comment by sergey48k** [comment]
You have a typo in output file name: .git -> .gif

**Comment by 4383** [comment]
Finely observed!
So, I have also added a check for output filename format and display en error if this use case appear again.
Fixed by #7 :)
